onnxscript/rewriter/llama_rule_sets.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+import onnxscript.ir as ir
+import onnxscript.rewriter.no_op as no_op
+import onnxscript.rewriter.pattern as orp
+from onnxscript.rewriter import pattern
+
+op = pattern.onnxop
+
+
+def transpose_identity(x, perm):
+    return op.Transpose(x, perm=perm)
+
+
+def transpose_identity_check(x: ir.Value, perm: ir.Attr | ir.RefAttr) -> bool:
+    if isinstance(perm, ir.RefAttr):
+        return False
+    if perm.type == ir.AttributeType.INTS:
+        if perm.value == list(range(len(perm.value))):
+            return True
+    return False
+
+
+def transpose_identity_rewrite(x: ir.Value, perm: ir.Attr):
```

Comment:
I think you need to add `op` as the first parameter here ... and not use the global `op`.

Sorry about the confusing notation ... I am also trying to unify the conventions so that `op` is the first parameter for both functions uniformly (WIP).